client/src/components/StatusForm.jsx

```diff
@@ -22,9 +22,9 @@ const StatusForm = ({ onSubmitChange, currStatus }) => {
     //need to change button options depending on whether the group has already been accepted or not
     return (
         <section className="change-status-form">
-            {currStatus === Status.PENDING?<p>You haven't responded to this invite yet.</p> : <p>Click "Drop" if you would like to drop this group.</p> }
-            {currStatus === Status.PENDING&&<button onClick={() => changeSelectedState(Status.ACCEPTED)}>Accept</button>}
-            {currStatus === Status.PENDING?<button onClick={() => changeSelectedState(Status.REJECTED)}>Ignore</button>:<button onClick={() => changeSelectedState(Status.DROPPED)}>Drop</button>}
+            {currStatus === Status.PENDING ? <p>You haven't responded to this invite yet.</p> : <p>Click "Drop" if you would like to drop this group.</p> }
+            {currStatus === Status.PENDING && <button onClick={() => changeSelectedState(Status.ACCEPTED)}>Accept</button>}
+            {currStatus === Status.PENDING ? <button onClick={() => changeSelectedState(Status.REJECTED)}>Ignore</button> : <button onClick={() => changeSelectedState(Status.DROPPED)}>Drop</button>}
```

Comment:
We are doing a ton of status === Status.PENDING CHECKS. Consider combining these into one check or extracting into another component